src/App.css

```diff
@@ -36,3 +36,11 @@
     transform: rotate(360deg);
   }
 }
+
+#movie-display {
+  display: grid;
+  grid-template-columns: 100px 100px 100px 100px 100px;
```

Comment:
Instead of hardcoding 5x100px columns, let's

- consistently use `em`-s everywhere so that we can scale everything by changing the font-size in body
- have it dynamically adjust number of columns by using `repeat(autofill ...)`

Reply:
Oh awesome, I did not know you could use ems in grid columns. Yes, lets fix that. I'll make an issue for it.